libs/blocks/quiz-entry/quizoption.js

```diff
@@ -144,7 +144,7 @@ export const GetQuizOption = ({
 
   return html`
   <div class="quiz-options-container" role="group" aria-labelledby="question" tabindex="0" onkeydown=${handleKey}>
-  ${index > 0 && html`<button onClick=${prev} class="carousel-arrow arrow-prev ${isRTL ? 'rtl' : ''}"></button>`}
+  ${index > 0 && html`<button onClick=${prev} class="carousel-arrow arrow-prev ${isRTL ? 'rtl' : ''}" aria-label="Previous"></button>`}
```

Comment:
Ideally, this should be translated as well for other locales

Reply:
Great suggestion! Of course we can work out translation like this, but it would be more involved because we would need to incorporate it into the strings spreadsheet. I'll have to write up a followup ticket for that, but there's a release deadline for US and they need these basic issues to be corrected first.